cmdline-pull/pubsub_sample.py

```diff
@@ -252,8 +252,12 @@ def pull_messages(client, args):
     """Pulls messages from a given subscription."""
     check_args_length(args, 3)
     subscription = get_full_subscription_name(args[0], args[2])
+    if len(args) > 3:
```

Comment:
How about to call the new variable `loop` or something, defaults to True? Also I don't think you need to modify the body.